src/Components/Web.JS/src/Platform/Mono/MonoPlatform.ts

```diff
@@ -122,6 +123,14 @@ export const monoPlatform: Platform = {
     return Module.getValue((baseAddress as any as number) + (fieldOffset || 0), 'i32');
   },
 
+  readUint64Field: function readHeapUint64(baseAddress: Pointer, fieldOffset?: number): number {
+    // Module.getValue(..., 'i64') doesn't work because the implementation treats 'i64' as
+    // being the same as 'i32'. Also we must take care to read both halves as unsigned.
+    const address = (baseAddress as any as number) + (fieldOffset || 0);
+    const heapU32Index = address >> 2;
+    return Module.HEAPU32[heapU32Index] + Module.HEAPU32[heapU32Index + 1] * uint64HighOrderShift;
+  },
```

Comment:
Do we need to check for overflow here?
Number.MAX_SAFE_NUMBER is (2**53 - 1) which we are never going to hit, but it might be good to have a check to detect errors?

Reply:
We could also file a bug to switch to BigInt when it becomes official. Chrome, (new Edge), and Firefox support it, and i'm sure there must be polyfills transforms for it if we want to be backwards compat.

Reply:
I’d rather not slow down every uint64 read for this. We know we’ll never reach 2^53 in normal use so it would only be useful for detecting some kind of memory corruption bug. If the memory is corrupted in even the slightest way, dozens of other things blow up.

Reply:
Actually I changed my mind. This particular function isn't specific to event handler IDs, but rather is just a general way of reading a uint64 value. So I agree it is valuable to have a way of noticing if we ever start trying to use it to read something it can't read (e.g., .NET's `uint64.MaxValue`).

Will add a check.